packages/appstore/src/components/account/account.tsx

```diff
@@ -1,54 +1,307 @@
-import React from 'react';
-import { Text } from '@deriv/components';
-import { Localize } from '@deriv/translations';
+import React, { useState } from 'react';
```

Comment:
we do not import react hooks directly in the deriv-app. please use `React.useState` instead